config/manifests/bases/dynatrace-monitoring.clusterserviceversion.yaml

```diff
@@ -0,0 +1,326 @@
+        path: args
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: 'Optional: List of environment variables to set for the installer'
+        displayName: OneAgent environment variable installer arguments
+        path: env
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: 'Optional: Adds additional labels for the OneAgent pods'
+        displayName: Labels
+        path: labels
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: 'Optional: If specified, indicates the pod''s priority. Name must be defined by creating a PriorityClass object with that name. If not specified the setting will be removed from the DaemonSet.'
+        displayName: Priority Class name
+        path: priorityClassName
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:PriorityClass
+      - description: 'Optional: Set custom proxy'
+        displayName: Proxy (static)
+        path: proxy.value
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:fieldGroup:Proxy
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:text
+      - description: 'Optional: set custom proxy settings from a secret with the field ''proxy'''
+        displayName: Proxy (from secret)
+        path: proxy.valueFrom
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:fieldGroup:Proxy
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:Secret
+      - description: 'Optional: set custom Service Account Name used with OneAgent pods'
+        displayName: Service Account name
+        path: serviceAccountName
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:ServiceAccount
+      - description: 'Optional: Adds custom RootCAs from a configmap'
+        displayName: TrustedCAs
+        path: trustedCAs
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:io.kubernetes:ConfigMap
+      - description: 'Optional: Defines the time to wait until OneAgent pod is ready after update - default 300 sec'
+        displayName: Wait seconds until ready
+        path: waitReadySeconds
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+        - urn:alm:descriptor:com.tectonic.ui:number
+      statusDescriptors:
+      - description: Dynatrace version being used.
+        displayName: Version
+        path: version
+        x-descriptors:
+        - urn:alm:descriptor:text
+      - description: The timestamp when the instance was last updated.
+        displayName: Last Updated
+        path: updatedTimestamp
+        x-descriptors:
+        - urn:alm:descriptor:text
+      version: v1alpha1
+  description: |
```

Comment:
Kubernetes and OpenShift have different descriptions.

Reply:
In the base file I can't provide two of them